dockerfile_parse/constants.py

```diff
@@ -14,3 +14,4 @@
 PY2 = version_info[0] == 2
 
 DOCKERFILE_FILENAME = 'Dockerfile'
+COMMENT_INSTRUCTION = 'COMMENT'
```

Comment:
Any reason not to make this `'#'`?

Reply:
yeah I about it but thought `'COMMENT'` was more self-explanatory. Either way it's arbitrary.